triage/2024-01-23.md

```diff
@@ -0,0 +1,259 @@
+probably just a blip.
+* [#116090](https://github.com/rust-lang/rust/pull/116090) caused a real regression on `doc`
+benchmarks, which was caused simply by adding a bunch of new methods to the standard library.
+
+Rollup of 8 pull requests [#120196](https://github.com/rust-lang/rust/pull/120196) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=a58ec8ff03b3269b20104eb7eae407be48ab95a7&end=6fff796eac247c072ddb84f8202bedecc8e94f0d&stat=instructions:u)
+
+| (instructions:u)                   | mean | range        | count |
+|:----------------------------------:|:----:|:------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 1.3% | [1.3%, 1.3%] | 1     |
+| Regressions ❌ <br /> (secondary)  | -    | -            | 0     |
+| Improvements ✅ <br /> (primary)   | -    | -            | 0     |
+| Improvements ✅ <br /> (secondary) | -    | -            | 0     |
+| All ❌✅ (primary)                 | 1.3% | [1.3%, 1.3%] | 1     |
+
+* Caused by [#120145](https://github.com/rust-lang/rust/pull/120145), which was a correctness fix,
+so marked as triaged.
+
+Rollup of 10 pull requests [#120242](https://github.com/rust-lang/rust/pull/120242) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=021861aea8de20c76c7411eb8ada7e8235e3d9b5&end=d5fd0997291ca0135401a39dff25c8a9c13b8961&stat=instructions:u)
+
+| (instructions:u)                   | mean | range        | count |
+|:----------------------------------:|:----:|:------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | -    | -            | 0     |
+| Regressions ❌ <br /> (secondary)  | 3.1% | [3.1%, 3.1%] | 1     |
+| Improvements ✅ <br /> (primary)   | -    | -            | 0     |
+| Improvements ✅ <br /> (secondary) | -    | -            | 0     |
+| All ❌✅ (primary)                 | -    | -            | 0     |
+
+* Caused by [#120137](https://github.com/rust-lang/rust/pull/120137), which added more validation
+to the compiler, and the perf. hit was deemed acceptable, so marked as triaged.
+
+#### Improvements
+
+Cache local DefId-keyed queries without hashing [#119977](https://github.com/rust-lang/rust/pull/119977) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=92f2e0aa62113a5f31076a9414daca55722556cf&end=098d4fd74c078b12bfc2e9438a2a04bc18b393bc&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.7%  | [0.7%, 0.7%]   | 1     |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -1.2% | [-8.2%, -0.2%] | 161   |
+| Improvements ✅ <br /> (secondary) | -1.5% | [-3.7%, -0.3%] | 64    |
+| All ❌✅ (primary)                 | -1.2% | [-8.2%, 0.7%]  | 162   |
+
+* Incredible wins across many benchmarks, caused by improving hashing in the compiler.
```

Comment:
Is it worth mentioning that icount wins were fantastic, but the wall-time wins were 3x better? That's a rare combination.

Reply:
Good point!

Reply:
Although, I'd also have to modify the TWiR PR, since it links to the old commit :sweat_smile: